### PR TITLE
METRON-1206: Make alerts UI conform to ops UI for install

### PR DESCRIPTION
**Pull request by merrimanr**
## Contributor Comments
This PR is a precursor to [METRON-1207](https://issues.apache.org/jira/browse/METRON-1207) and [METRON-1208](https://issues.apache.org/jira/browse/METRON-1208).  The list of changes include:
- addition of a configuration file (as opposed to passing in arguments to the start script)
- a new init script for start/stop/restart operations
- updated assembly.xml to reflect shared expressjs directory structure
- changed port for Alerts UI to 4201 to avoid conflict with Management UI
- updated Alerts UI documentation

This can be tested by following the instructions in the README.  The previously mentioned follow-on PRs are also ready once this has been accepted.

## Pull Request Checklist

Thank you for submitting a contribution to Apache Metron.  
Please refer to our [Development Guidelines](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=61332235) for the complete guide to follow for contributions.  
Please refer also to our [Build Verification Guidelines](https://cwiki.apache.org/confluence/display/METRON/Verifying+Builds?show-miniview) for complete smoke testing guides.  

In order to streamline the review of the contribution we ask you follow these guidelines and ask you to double check the following:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? If not one needs to be created at [Metron Jira](https://issues.apache.org/jira/browse/METRON/?selectedTab=com.atlassian.jira.jira-projects-plugin:summary-panel). 
- [x] Does your PR title start with METRON-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [x] Has your PR been rebased against the latest commit within the target branch (typically master)?

### For code changes:
- [x] Have you included steps to reproduce the behavior or problem that is being changed or addressed?
- [x] Have you included steps or a guide to how the change may be verified and tested manually?
- [x] Have you ensured that the full suite of tests and checks have been executed in the root metron folder via:
  ```
  mvn -q clean integration-test install && build_utils/verify_licenses.sh 
  ```

- [x] Have you written or updated unit tests and or integration tests to verify your changes?
- [x] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [x] Have you verified the basic functionality of the build by building and running locally with Vagrant full-dev environment or the equivalent?

### For documentation related changes:
- [x] Have you ensured that format looks appropriate for the output in which it is rendered by building and verifying the site-book? If not then run the following commands and the verify changes via `site-book/target/site/index.html`:

  ```
  cd site-book
  mvn site
  ```

#### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.
It is also recommended that [travis-ci](https://travis-ci.org) is set up for your personal repository such that your branches are built there before submitting a pull request.

### CODE REVIEW

File: metron-interface/metron-alerts/README.md [hunk 1]
```diff
@@ -6,15 +6,74 @@
...
+    web
+      expressjs
+        alerts-server.js
+      alerts-ui
+        web assets (html, css, js, ...)
+    ```
+
+1. Copy the `$METRON_HOME/bin/metron-alerts-ui` script to `/etc/init.d/metron-alerts-ui`
+
+1. [Express](https://expressjs.com/) is installed at `$METRON_HOME/web/expressjs/` as part of the Management UI installation process.  The Management UI should be installeed first on the same host as the Alerts UI.
```
Comment (justinleet) [inline metron-interface/metron-alerts/README.md]: `installeed` to `installed`.

File: metron-interface/metron-alerts/README.md [hunk 2]
```diff
@@ -6,15 +6,74 @@
...
+
+### From Package Manager
```
Comment (justinleet) [inline metron-interface/metron-alerts/README.md]: RPM sounds like it exists in one of the followon PRs that's coming.  Can we move this section to that PR?

File: metron-interface/metron-alerts/README.md [hunk 3]
```diff
@@ -6,15 +6,74 @@
...
+
+## Usage
+
+After configuration is complete, the Management UI can be managed as a service:
+
+```
+service metron-alerts-ui start
+```
+
+The application will be available at http://host:4201 assuming the port is set to `4201`.  Logs can be found at `/var/log/metron/metron-management-ui.log`.
```
Comment (justinleet) [inline metron-interface/metron-alerts/README.md]: `metron-management-ui.log` to `metron-alerts-ui.log`

File: metron-interface/metron-alerts/README.md [hunk 4]
```diff
@@ -28,7 +87,7 @@
     ```
 1. You can view the GUI @http://localhost:4200 . The default credentials for login are admin/password
```
Comment (justinleet) [inline metron-interface/metron-alerts/README.md]: Should be 4201 now, right?

**Comment by merrimanr** [comment]
The latest commit should address all the PR comments.

File: metron-interface/metron-alerts/README.md (hunk 4 above)
Comment (justinleet) [reply metron-interface/metron-alerts/README.md]: @merrimanr Looks like Github hid this one since it's a change to a preexisting thing.

Comment (justinleet) [reply metron-interface/metron-alerts/README.md]: Nevermind, github is is being weird for me and showing it inconsistently

**Comment by justinleet** [comment]
+1, thanks for making the adjustments

**Pull request closed without merging**